.github/workflows/deploy-book.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         cp --recursive --dereference --no-preserve=mode,ownership result/share/doc/nedryglot/manual dist
 
     - name: Upload Artifact
-      uses: actions/upload-pages-artifact@v2
+      uses: actions/upload-pages-artifact@v4
```

Comment:
I think it should be 3?

https://github.com/actions/upload-pages-artifact/releases/tag/v3.0.1 looks to be the latest

Reply:
Yeah I dont know where I got v4 from, thought I copied it from docs but must have dreamt up something